src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/AutoInsert/AttributeSnippetFormatOnAutoInsertProviderTest.cs

```diff
@@ -86,7 +83,7 @@ public void OnTypeEqual_AfterTagHelperTag_Noops()
 @addTagHelper *, TestAssembly
 
 <section>
-    <span|></span>
+    <span=$$></span>
```

Comment:
Interesting, why'd you have to change the structure of the test to include the `=`?

Reply:
Previously the test would insert a character at the `|` location. Now it combines the two.

Reply:
Isn't that a different test though? In this specific test it's trying to validate that the act of typing `=` has a specific result

Reply:
➡️ See https://github.com/dotnet/aspnetcore-tooling/pull/3073#discussion_r571204257

Reply:
oooo I see. Alright really good to know. Thanks for clarifying and apologies for not seeing the other comment.

Reply:
> apologies for not seeing the other comment.

I just added the other comment for the purposes of adding my explanation 😂